src/main/java/org/dnd/timeet/common/security/CustomUserDetails.java

```diff
@@ -28,14 +38,19 @@ public Collection<? extends GrantedAuthority> getAuthorities() {
 
     @Override
     public String getPassword() {
-        return user.getPassword();
+        return null;
```

Comment:
현재는 Override의 구현을 위해 return null을 하였으나, 추후 사용자가 모르고 해당 메서드를 사용했을 때 트러블 슈팅이 어려울 수 있어, Exception 등을 발생하는 처리가 필요할 것 같습니다.

Reply:
좋아요 ! 아래와 같이 구현하면 되겠네용 ! 😁

```suggestion
        return throw new UnsupportedOperationException("getPassword 메서드는 사용되지 않습니다.");
```

Reply:
마침 UnsupportedOperationException이 있었군요! 알려주셔서 감사해요 ☺️ 반영해서 커밋 올려두겠습니다!